bin/run_docker.py

```diff
@@ -267,6 +298,44 @@ def mount_volumes() -> dict:
     return volumes
 
 
+def validate_submission(
```

Comment:
Just a thought experiment but to make this script "easier" less complicated, we could have a nextflow step that validate the docker container so that we know for sure that what's provided to run_docker step can be executed. That said, that may be overkill.

Reply:
+1.

IIRC, @thomasyu888 had created a challengeutils command for this. `validate-docker` I think?

```
challengeutils validate-docker SUBMISSION_ID [--output JSON]
```

Reply:
Thanks! Made a [ticket](https://sagebionetworks.jira.com/browse/ORCA-327) to tackle this after PEGS!